monitoring/custom-checks/in-servicepulse.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Managing custom checks in ServicePulse
+summary: Describes how ServicePulse monitors custom check activity
+reviewed: 2018-01-05
+redircts:
+  - servicepulse/intro-endpoints-custom-checks
+---
+
+ServicePulse monitors the health and activity of an NServiceBus endpoint using [Heartbeats](/monitoring/heartbeats/) and [Custom Checks](/monitoring/custom-checks/).
+
+The main dashboard shows a custom checks icon which will indicate if there are any failing custom checks.
```

Comment:
Heading *Dashboard*?

Reply:
If we add a heading here, would we need to have a separate heading for the details page? These would be very short sections.